sdk/PowerBI.Core/Security/PowerBIToken.cs

```diff
@@ -70,29 +87,69 @@ public PowerBIToken()
         /// <param name="workspaceCollectionName">The workspace collection name</param>
         /// <param name="workspaceId">The workspace id</param>
         /// <param name="reportId">The report id</param>
+        /// <param name="datasetId">The dataset id</param>
         /// <param name="username">The RLS username</param>
         /// <param name="roles">The RLS roles</param>
+        /// <param name="scopes">The permission scopes</param>
         /// <returns>The Power BI access token</returns>
-        public static PowerBIToken CreateReportEmbedToken(string workspaceCollectionName, string workspaceId, string reportId, string username = null, IEnumerable<string> roles = null)
+        public static PowerBIToken CreateReportEmbedToken(string workspaceCollectionName, string workspaceId, string reportId = null, string datasetId = null, string username = null, IEnumerable<string> roles = null, string scopes = null)
         {
             var expires = DateTime.UtcNow.Add(TimeSpan.FromSeconds(DefaultExpirationSeconds));
-            return CreateReportEmbedToken(workspaceCollectionName, workspaceId, reportId, expires, username, roles);
+            return CreateReportEmbedToken(workspaceCollectionName, workspaceId, reportId: reportId, datasetId: string.Empty, expiration: expires, username: username, roles: roles, scopes: scopes);
```

Comment:
Why do we define datasetId as a parameter if it's being ignored later? Please change the signature accordingly